webmanifest.json

```diff
@@ -24,7 +24,7 @@
             "type": "image\/png"
         },
         {
-            "src": "/static/icons/maskable-icon-640x640.png",
+            "src": "/static/icons/maskable-icon-213x213.png",
             "sizes": "640x640",
```

Comment:
Should size be "213x213"?

Reply:
That is what the pwabuilder built. These icons are not final. I will swap them once everything works.